packages/template-retail-react-app/app/ssr.js

```diff
@@ -45,7 +45,7 @@ const options = {
     // Set this to false if using a SLAS public client
     // When setting this to true, make sure to also set the PWA_KIT_SLAS_CLIENT_SECRET
     // environment variable as this endpoint will return HTTP 501 if it is not set
-    useSLASPrivateClient: false
+    useSLASPrivateClient: true
```

Comment:
Same for this line. Doing this will make private client the default for our local development environment.

I think we want to do this, but we'll need to coordinate with our team on setting the client secret in our environments